dotcom-rendering/src/components/TagFrontHeader.tsx

```diff
@@ -0,0 +1,322 @@
+import { css, jsx } from '@emotion/react';
+import {
+	breakpoints,
+	from,
+	headline,
+	palette,
+	space,
+	until,
+} from '@guardian/source-foundations';
+import { Fragment, type ReactNode } from 'react';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+import { isElement, parseHtml } from '../lib/domUtils';
+import { logger } from '../server/lib/logging';
+import type { DCRContainerPalette } from '../types/front';
+import { ContainerTitle } from './ContainerTitle';
+import { generateSources, Sources } from './Picture';
+
+type Props = {
+	title: string;
+	description?: string;
+	image?: string;
+	containerPalette?: DCRContainerPalette;
+};
+
+const width = (columns: number, columnWidth: number, columnGap: number) =>
+	`width: ${columns * columnWidth + (columns - 1) * columnGap}px;`;
+
+/** Not all browsers support CSS grid, so we set explicit width as a fallback */
+const fallbackStyles = css`
+	@supports not (display: grid) {
+		padding: 0 12px;
+		margin: 0 auto;
+
+		${from.mobileLandscape} {
+			padding: 0 20px;
+		}
+
+		${from.tablet} {
+			${width(12, 40, 20)}
+		}
+
+		${from.desktop} {
+			${width(12, 60, 20)}
+		}
+
+		${from.leftCol} {
+			${width(14, 60, 20)}
+		}
+
+		${from.wide} {
+			${width(16, 60, 20)}
+		}
+	}
+`;
+
+const containerStyles = css`
+	display: grid;
+
+	grid-template-rows:
+		[headline-start image-start] auto
+		[headline-end content-start] auto
+		[content-end image-end] auto;
+
+	grid-template-columns:
+		[viewport-start] 0px
+		[content-start title-start]
+		repeat(3, minmax(0, 1fr))
+		[content-end title-end image-start]
+		minmax(0, 1fr)
+		[image-end]
+		0px [viewport-end];
+
+	grid-auto-flow: dense;
+	column-gap: 10px;
+
+	${from.mobileLandscape} {
+		column-gap: 20px;
+	}
+
+	${from.tablet} {
+		grid-template-columns:
+			[viewport-start] minmax(0, 1fr)
+			[content-start title-start]
+			repeat(6, 40px) [content-end title-end]
+			repeat(4, 40px)
+			[image-start]
+			repeat(2, 40px)
+			[image-end]
+			minmax(0, 1fr) [viewport-end];
+	}
+
+	${from.desktop} {
+		grid-template-columns:
+			[viewport-start] minmax(0, 1fr)
+			[content-start title-start]
+			repeat(6, 60px) [content-end]
+			repeat(4, 60px)
+			[title-end image-start]
+			repeat(2, 60px)
+			[image-end]
+			minmax(0, 1fr) [viewport-end];
+	}
+
+	${from.leftCol} {
+		grid-template-rows:
+			[headline-start content-start image-start] auto
+			[headline-end content-end image-end] auto;
+
+		grid-template-columns:
+			[viewport-start] minmax(0, 1fr)
+			[title-start]
+			repeat(2, 60px)
+			[title-end content-start]
+			repeat(7, 60px) [content-end]
+			repeat(3, 60px)
+			[image-start]
+			repeat(2, 60px)
+			[image-end]
+			minmax(0, 1fr) [viewport-end];
+	}
+
+	${from.wide} {
+		grid-template-columns:
+			[viewport-start] minmax(0, 1fr)
+			[title-start]
+			repeat(3, 60px)
+			[title-end content-start]
+			repeat(8, 60px) [content-end]
+			repeat(2, 60px)
+			[ image-start]
```

Comment:
Does this space do anything?

```suggestion
			[image-start]
```

Reply:
Don't think so - have removed!